README.md

```diff
@@ -4,7 +4,7 @@
 ![CI](https://github.com/yannikschaelte/yaml2sbml/workflows/CI/badge.svg)
 [![codecov](https://codecov.io/gh/yannikschaelte/yaml2sbml/branch/master/graph/badge.svg?token=AORT3GREQJ)](https://codecov.io/gh/yannikschaelte/yaml2sbml)
```

Comment:
Is it intended that these links are to forks or previous locations of the repository? Unable to select all links in this "review" comment, there's one more 1 line above.

@martamatos @yannikschaelte

Reply:
There are also some `utm` parameters that maybe could be removed.

Reply:
I should have updated the old links to the new location in https://github.com/yaml2sbml-dev/yaml2sbml/pull/81/files already.

Reply:
Nope, some links in there are obsolete. Marta and me had a discussion in #55 and decided to merge the rewrite of the README after the "you can pip install the package is done". Will do this today :)

Reply:
Will leave as-is for this PR then